lib/HTML/FormBuilder/Select.pm

```diff
@@ -63,8 +65,12 @@ has name => (
 has options => (
     is  => 'rw',
     isa => sub {
-        die "$_[0] is not ArrayRef[HashRef[Any]]"
-            unless (ref($_[0]) eq 'ARRAY' && ref($_[0][0]) eq 'HASH');
+        my $arg = $_[0];
+        my $msg = Dumper($arg) . " is not ArrayRef[HashRef[Any]]";
```

Comment:
I'd probably change this description to match what you're actually testing: `ArrayRef[Maybe[HashRef]]1